brewtils/plugin.py

```diff
@@ -248,9 +248,9 @@ def client(self, new_client):
 
         # Several _system properties can come from the client, so update if needed
         if not self._system.name:
-            self._system.name = getattr(new_client, "_bg_name")
+            self._system.name = getattr(new_client, "_bg_name", None)
         if not self._system.version:
-            self._system.version = getattr(new_client, "_bg_version")
+            self._system.version = getattr(new_client, "_bg_version", None)
```

Comment:
The test was failing when at the locations changed. TheBurchLog wanted test updated so PR would pass the test. It failed because hidden was not in fixtures for command dictionary. The other changes in test because test found errors in the changed locations referenced below.

Reply:
So just to be clear, this test was failing before you added the `hidden` parameter to the test fixture? Now that you've added that parameter to the fixture, is this change still necessary?